sheets/utils.py

```diff
@@ -522,6 +337,19 @@ def spreadsheet_repr(spreadsheet=None, spreadsheet_metadata=None):
     return "'{}', id: {}".format(title, sheet_id)
 
 
+def clean_sheet_value(value):
+    """
+    Takes a spreadsheet cell value and returns a cleaned version
+
+    Args:
+        value (str): A raw spreadsheet cell value
+
+    Returns:
+        str or None: A string with whitespace stripped, or None if the resulting value was an empty string
+    """
+    return value.strip() if value else None
```

Comment:
If `value` is `" "`, do we want the returned value to be `""` or `None`? Is that something that ever comes up in the input data?

Reply:
Noticed this in another branch and forgot to update here. Great catch